lib/modules/manager/cargo/extract.spec.ts

```diff
@@ -112,6 +113,19 @@ describe('modules/manager/cargo/extract', () => {
       expect(res?.deps).toHaveLength(3);
     });
 
+    it('extracts workspace dependencies', async () => {
+      const res = await extractPackageFile(cargo7toml, 'Cargo.toml', config);
```

Comment:
please reduce to minimum and inline fixture for io perf.

Reply:
I've inlined them, It introduces a bit more verbosity, but please have a look and tell me what you think =D